src/nat/llm/aws_bedrock_llm.py

```diff
@@ -33,11 +36,14 @@ class AWSBedrockModelConfig(LLMBaseConfig, RetryMixin, name="aws_bedrock"):
     model_name: str = Field(validation_alias=AliasChoices("model_name", "model"),
                             serialization_alias="model",
                             description="The model name for the hosted AWS Bedrock.")
-    temperature: float = Field(default=0.0, ge=0.0, le=1.0, description="Sampling temperature in [0, 1].")
-    max_tokens: int | None = Field(default=1024,
-                                   gt=0,
-                                   description="Maximum number of tokens to generate."
-                                   "This field is ONLY required when using AWS Bedrock with Langchain.")
+
+    temperature: float = OptimizableField(default=0.0,
+                                          description="Sampling temperature in [0, 1].",
+                                          space=SearchSpace(high=0.9, low=0.1, step=0.2))
```

Comment:
Some models don't support temperature. Is it possible to still have:

* `temperature: float | None`
* `default=None`
* `space=SearchSpace(high=0.9, low=0.1, step=0.2)`

and only have the search space explored IFF temperature is set to non-`None` ?